db/seeds.rb

```diff
@@ -53,6 +53,5 @@
     )
   end
 
-  subject_csv = Rails.root.join("config", "subjects.csv")
-  SubjectImporter.new(subject_csv).run
+  Rake::Task["subjects:import"].invoke("2017FA")
```

Comment:
I'm not crazy about this either, but I think it's relatively harmless. Seeding with old data wouldn't be bad -- we can run the rake task again with the correct term.

That said, I wonder if we can be smart about it somehow. Obviously, we can get the current year, maybe we can also divide the months up into SP/JA/SU/FA semesters?